pontoon/base/models.py

```diff
@@ -25,12 +25,101 @@
 log = logging.getLogger('pontoon')
 
 
+class UserTranslationsManager(models.Manager):
+    """
+    Provides various method to interact with larger sets of translations and their stats for user.
+    """
+
+    def _changed_translations_counts(self, start_date=None, end_date=None, filter=None, exclude=None):
+        """
+        Helper method, returns expression object which allows us to annotate querysets
+        with counts of translations.
+        Method supports exclusion of fields, you can exclude field if param contains
```

Comment:
This is incorrect. Also, instead of having `exclude` and `include` parameters, maybe just accept `Q` objects as `*args` and allow the caller to determine whether they include or exclude things? It'd simplify this function a bit.